drake/examples/Quadrotor/runLQR.cpp

```diff
@@ -1,8 +1,10 @@
 #include <iostream>
-#include "Quadrotor.h"
+#include "drake/examples/Quadrotor/Quadrotor.h"
 #include "drake/systems/Simulation.h"
 #include "drake/systems/controllers/LQR.h"
 #include "drake/systems/plants/BotVisualizer.h"
+#include "drake/systems/cascade_system.h"
```

Comment:
i presume that's the naming convention from the style guide? (which you can see is different than all the existing files :)